in_toto/verifylib.go

```diff
@@ -679,7 +752,12 @@ func InTotoVerify(layoutMb Metablock, layoutKeys map[string]Key,
 		return summaryLink, err
 	}
 
-	// TODO: Substitute parameters
+	if len(parameterDictionary) > 0 {
```

Comment:
Minor nit: I think this check could be performed inside `SubstituteParameters`. It would make `InTotoVerify` slimmer. :)